### PR TITLE
Delete button confirm

### PR DESCRIPTION
**Pull request by lgarcia2689**
https://www.loom.com/share/7d40711129674ea5a2eca1083ba374cb

# Description
Created Modal for delete button, now when clicking on delete button, instead of deleting data immediately, modal opens asking if user is sure and given the option to cancel or continue deletion

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/AdminDashboard/CompleteIncident.js
Comment (ooladuwa) [inline src/components/AdminDashboard/CompleteIncident.js:2]: Used existing Modal component from antd to negate redundancy of creating nearly identical modal from scratch.

**Review by ooladuwa — APPROVED** [review]
Great job overall. We worked to ensure that the delete functionality was a two-stage process. Delete button appearance is red to convey danger while confirmation fo delete follows the style conventions of the rest of the app. Tested deletion from confirmed and unconfirmed incidents. Looks good. Approve.

File: src/components/timeline/TimelineItems.js
Comment (kcjerrell) [inline src/components/timeline/TimelineItems.js:74]: were any changes made here, or was it just reformatted?

**Review by kcjerrell — APPROVED** [review]
I appreciate using an existing component instead of creating a new one, and I think I like the modal better than having the buttons switch to "confirm/cancel." I might do that with the approve/disapprove buttons too.

File: src/components/timeline/TimelineItems.js
Comment (lgarcia2689) [reply src/components/timeline/TimelineItems.js:74]: It was just reformatting

File: src/components/AdminDashboard/CompleteIncident.js
Comment (Ethan-Edmond) [inline src/components/AdminDashboard/CompleteIncident.js]: Why does this promise reject based on a random number?

Comment (kcjerrell) [reply src/components/AdminDashboard/CompleteIncident.js]: To keep you on your toes.

**Review by Ethan-Edmond — DISMISSED** [review]
Ok, I finished my review. I'll approve once you remove the random promise from the onOk of your confirm. If you need a promise returned, use your axios call instead.

File: src/components/AdminDashboard/CompleteIncident.js
Comment (austin-carman) [inline src/components/AdminDashboard/CompleteIncident.js:66]: Is this a confirmation message to the user that appears after the return (where setTimeout is being called)? If so, does this need to be changed to something like "Incident has been deleted"?

**Review by austin-carman — CHANGES_REQUESTED** [review]
Requesting change for the comment I left in the CompleteIncident.js file. Ready to approve after this change - if I'm wrong about the purpose of this line, please let me know and I'll be ready to approve without any changes (assuming Ethan's change request about the Promise has also been met)

File: src/components/AdminDashboard/CompleteIncident.js
Comment (ooladuwa) [reply src/components/AdminDashboard/CompleteIncident.js:66]: This is a confirmation message to the user that appears after they click the delete button(feature added in the subsequent pull request)

Comment (ooladuwa) [reply src/components/AdminDashboard/CompleteIncident.js:66]: It is not a confirmation message. It appears when the user clicks delete. It adds a step. We decided not to include an additional confirmation message.

Comment (austin-carman) [reply src/components/AdminDashboard/CompleteIncident.js:66]: Oh ok, I see. It seems a little weird to tell the user that, maybe could include more details about the incident that's about to get deleted? But that's just my personal preference and functionality is good so I'll approve

**Review by austin-carman — APPROVED** [review]
Naj cleared up what the content message is - personal preference might be to include more details about the incident in question instead but functionality is good so approved

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This branch is now so out-of-date, we'll need to hold a session between the developer, some peers, and myself, the Release Manager for this product to resolve the merge conflicts.

File: src/components/AdminDashboard/CompleteIncident.js
Comment (Ethan-Edmond) [inline src/components/AdminDashboard/CompleteIncident.js:78]:
```suggestion
        return oktaAxios
          .delete(`/dashboard/incidents/${incident.incident_id}`)
          .then(res => {
            window.location.reload();
          })
          .catch(err => {
            console.log(err);
          });
```

**Review by Ethan-Edmond — CHANGES_REQUESTED** [review]
Previous changes did not address my concern. I discussed it with Luis. The attached change would address my concern.

**Comment by kcjerrell** [comment]
I am pretty sure this PR can be closed. Subsequent PRs have addressed this issue.

**Pull request closed without merging**
